src/index.js

```diff
@@ -1,15 +1,18 @@
+/* eslint-disable no-await-in-loop */
+
 import './style.css';
 import '@fortawesome/fontawesome-free/js/all.js';
 import pokemonCard from './modules/pokemonCard.js';
-/* eslint-disable no-await-in-loop */
+import { addPopUp } from './modules/comments-popup.js';
 
-const pokedex = 157;
+const pokedex = 10;
```

Comment:
pokedex is declared in both the index file and the comment popup.js file. kindly export from the comments-popup file

Reply:
Hi @Emmykage 😁 
I solved this issue in the next [PR](https://github.com/mikemtzp/PokemonAPI/pull/24)